charts/kubernetes/templates/topology.yaml

```diff
@@ -78,12 +105,16 @@ spec:
               tagSelector: 'cluster={{.Values.clusterName}}'
             display:
               expr: |
-                dyn(results).filter(r, !(r.name.startsWith('junit-'))).map(r, {
+                dyn(results).filter(r, has(r.config.metadata.ownerReferences) && r.config.metadata.ownerReferences[0].kind == 'ReplicaSet').map(r, {
                   'name': r.name,
                   'namespace': r.tags['namespace'],
                   'type': 'Kubernetes::Pod',
                   'external_id': r.tags['namespace'] + '/' + r.name,
-                  'parentLookup': {'name': r.tags['namespace'], 'type': 'Kubernetes::Namespace'},
+                  'parentLookup': {
```

Comment:
I don't think we want to remove the Namespace -> Pod, The pod should be in both locations